tests/test_cli.py

```diff
@@ -312,3 +313,23 @@ def test_check_region_aggregation(tmp_path):
         )
     )
     assert_iamframe_equal(IamDataFrame(tmp_path / "results.xlsx"), exp_result)
+
+
+def test_cli_export_to_excel(simple_definition, tmpdir):
```

Comment:
Looks like this variable is not needed
```suggestion
def test_cli_export_to_excel(tmpdir):
```

Reply:
Thanks for catching that.